### ISSUE TITLE
Request: Do Not Show Any GUI Alerts in Silent Mode

### ISSUE THREAD

**Issue by PicoMitchell** [rcmaehl/WhyNotWin11#595]
When running WhyNotWin11 within Windows 11 (which I was doing for various testing), I noticed that I'm seeing the "You're running the latest build" alert with a "Not Supported" title even when running in `/silent` mode.

I'm hoping this can be adjusted so that no alerts of any kind show up when running in `/silent` mode since I think that would be the general expectation when using this argument. I'm not sure what other alerts are possible, but I think that if there is critical issue or alert when running in `/silent` it should be returned from the executable on the command line rather than a GUI prompt (or simply not shown at all if it's not a critical issue).

**Comment by rcmaehl** [comment]
Hi @PicoMitchell

Please try the latest dev build. https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip
Additionally in regards to #594, please try the `/force` flag

**Comment by PicoMitchell** [comment]
Thanks, I've finally got a chance to test!

Here is what I'm seeing on Win 11 WinPE:

`WhyNotWin11.exe /export TXT X:\WNW11.txt /silent` just fails to launch a process with no error, presumably because of the version check failure.

`WhyNotWin11.exe /export TXT X:\WNW11.txt /silent /force` launches a process but hangs forever (waited longer than the 2 minute timeout). Assuming this hang still has something to do with GPU drivers not being available. I will post over on that issue with more findings on that behavior.

But, I think the `/force` argument will satsify the need to be able to run in `/silent` mode with no prompts. I also confirmed that `/force` does not show the version error when run without `/silent`. Thank you!
